src/Map/assets/dist/abstract_map_controller.js

```diff
@@ -74,7 +74,7 @@ class default_1 extends Controller {
         return ({ definition }) => {
             this.dispatchEvent(eventBefore, { definition });
             const drawing = factory({ definition });
-            this.dispatchEvent(eventAfter, { [type]: drawing });
+            this.dispatchEvent(eventAfter, { [type]: drawing, definition: definition });
```

Comment:
Since I can't push on your repository, you will need to do the following changes:
- This change must be done in the `src/abstract_map_controller.ts` file
- You also need to update the `info-window:after-create` event
- finally, you have to rebuild the dist files (https://github.com/symfony/ux/blob/2.x/CONTRIBUTING.md#working-with-assets)
Thanks!

Reply:
@Kocal I pushed some changes.